docs/modules/ROOT/pages/getting-started.adoc

```diff
@@ -237,64 +237,23 @@ Go::
 +
 [source,go]
 ----
-package main
-
-import (
-	"context"
-	"fmt"
-	"log"
-
-	"github.com/hazelcast/hazelcast-go-client"
-)
-
-// Installation note: Run following command
-// go get github.com/hazelcast/hazelcast-go-client
-
-func main() {
-	ShutdownProperly := func(ctx context.Context, hz *hazelcast.Client, err error) {
-		errSd := hz.Shutdown(ctx)
-		switch {
-		case err != nil && errSd == nil:
-			log.Fatal(fmt.Errorf("%v", err))
-		case err != nil && errSd != nil:
-			log.Fatal(fmt.Errorf("%v; %v", err, errSd))
-		}
-	}
-
-	ctx := context.TODO()
-
-	// Create a new Hazelcast client
-	hz, err := hazelcast.StartNewClient(ctx)
-	if err != nil {
-		log.Fatalln(err)
-	}
+//Installation notes: Run the following command
+//go get github.com/hazelcast/hazelcast-go-client
 
-	// Create a Distributed Map in the cluster
-	mp, err := hz.GetMap(ctx, "myDistributedMap")
-	if err != nil {
-		ShutdownProperly(ctx, hz, err)
-	}
-
-	// Standard Put and Get
-	_, err = mp.Put(ctx, "1", "John")
-	if err != nil {
-		ShutdownProperly(ctx, hz, err)
-	}
-	_, err = mp.Put(ctx, "2", "Mary")
-	if err != nil {
-		ShutdownProperly(ctx, hz, err)
-	}
-	_, err = mp.Put(ctx, "3", "Jane")
-	if err != nil {
-		ShutdownProperly(ctx, hz, err)
-	}
+import "github.com/hazelcast/hazelcast-go-client"
 
+func mapSampleRun() {
+	// Start the client and connect to the cluster
+	hz, _ := hazelcast.NewClient()

```

Comment:
```suggestion
	client, err := hazelcast.NewClient()

```
"client" as client variable is more clear I think. Also it would be great if we handle errors in these examples 🙏. They are also part of the public API.

Reply:
Sorry did not notice this was revert. Please ignore